packages/client/src/navigation/index.ts

```diff
@@ -180,7 +181,9 @@ export function goToApplicationConfig() {
 }
 
 export function goToHomeTab(tabId: IWORKQUEUE_TABS, selectorId = '') {
-  return push(formatUrl(REGISTRAR_HOME_TAB, { tabId, selectorId }))
+  return push(formatUrl(REGISTRAR_HOME_TAB, { tabId, selectorId }), {
+    from: 'correction summary page'
```

Comment:
If we doing the redirect functionality, these state work is not needed... could you please undo this?

Reply:
alright.